Panic when calling plan complete command

### Description

Happens when calling `gravity plan complete` for a hung join operation.

```
[root@ip-10-218-32-101 ~]# gravity plan complete --operation-id=da8d0b02-69ff-4e9d-8d73-fd28f9fdbd85
Wed Oct 14 22:20:17 UTC	Connecting to agent
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x27406ff]
goroutine 1 [running]:
github.com/gravitational/gravity/lib/clients.(*AuthClient).EmitAuditEvent(0x0, 0x39fd1f1, 0x10, 0x39e0f30, 0x6, 0xc0007d9b90, 0x39deace, 0x39deafe)
	<autogenerated>:1 +0x2f
github.com/gravitational/gravity/lib/ops/opsservice.(*Operator).EmitAuditEvent(0xc001090000, 0x4139f60, 0xc0000d0028, 0xc0018a9800, 0x24, 0xc001578280, 0x11, 0x39fd1f1, 0x10, 0x39e0f30, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/opsservice/service.go:1416 +0x19b
github.com/gravitational/gravity/lib/ops/events.emit(0x4139f60, 0xc0000d0028, 0x41ead20, 0xc001090000, 0x39fd1f1, 0x10, 0x39e0f30, 0x6, 0xc0007d9b90, 0x327d760, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/events/events.go:66 +0x208
github.com/gravitational/gravity/lib/ops/events.Emit(0x4139f60, 0xc0000d0028, 0x41ead20, 0xc001090000, 0x39fd1f1, 0x10, 0x39e0f30, 0x6, 0xc0001567b8, 0x1, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/events/events.go:42 +0x1dc
github.com/gravitational/gravity/lib/ops/events.EmitForOperation(0x4139f60, 0xc0000d0028, 0x41ead20, 0xc001090000, 0xc0013d4d50, 0x24, 0xc0013d4d80, 0x24, 0xc000d314e0, 0x11, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/events/events.go:54 +0x191
github.com/gravitational/gravity/lib/ops/opsservice.(*operationGroup).emitAuditEvent(0xc000b8fc20, 0x4139f60, 0xc0000d0028, 0xc0013d4d50, 0x24, 0xc0013d4d80, 0x24, 0xc000d314e0, 0x11, 0xc0013646b0, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/opsservice/operationgroup.go:124 +0x166
github.com/gravitational/gravity/lib/ops/opsservice.(*operationGroup).compareAndSwapOperationState(0xc000b8fc20, 0x4139f60, 0xc0000d0028, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc0013d4390, 0x24, 0x0, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/opsservice/operationgroup.go:333 +0x39c
github.com/gravitational/gravity/lib/ops/opsservice.(*site).compareAndSwapOperationState(0xc000480280, 0x4139f60, 0xc0000d0028, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc0013d4390, 0x24, 0x0, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/opsservice/site.go:466 +0xbe
github.com/gravitational/gravity/lib/ops/opsservice.(*Operator).SetOperationState(0xc001090000, 0x4139f60, 0xc0000d0028, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc0013d4390, 0x24, 0x39e1cc2, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/opsservice/service.go:1050 +0x26a
github.com/gravitational/gravity/lib/ops.FailOperation(0x4139f60, 0xc0000d0028, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc0013d4390, 0x24, 0x40b3b80, 0xc001090000, ...)
	/gopath/src/github.com/gravitational/gravity/lib/ops/utils.go:289 +0x253
github.com/gravitational/gravity/tool/gravity/cli.completeJoinPlanFromExistingNode(0xc000bd9520, 0xc0013d4390, 0x24, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc001169830, 0x10, 0xca7d261, ...)
	/gopath/src/github.com/gravitational/gravity/tool/gravity/cli/install.go:591 +0x342
github.com/gravitational/gravity/tool/gravity/cli.completeJoinPlanForOperation(0xc000bd9520, 0xc0013d4390, 0x24, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc001169830, 0x10, 0xca7d261, ...)
	/gopath/src/github.com/gravitational/gravity/tool/gravity/cli/install.go:571 +0x108
github.com/gravitational/gravity/tool/gravity/cli.completeOperationPlanForOperation(0xc000bd9520, 0x4109ca0, 0xc0002f8c00, 0xc0013d4390, 0x24, 0xc0013d43c0, 0x24, 0xc0012faa20, 0x11, 0xc001169830, ...)
	/gopath/src/github.com/gravitational/gravity/tool/gravity/cli/operation.go:242 +0x47b
github.com/gravitational/gravity/tool/gravity/cli.completeOperationPlan(0xc000bd9520, 0x4109ca0, 0xc0002f8c00, 0x7fff9480e6e7, 0x24, 0xc0004aab70, 0xc000000244)
	/gopath/src/github.com/gravitational/gravity/tool/gravity/cli/operation.go:230 +0xf5
github.com/gravitational/gravity/tool/gravity/cli.Execute(0xc0002f8c00, 0xc000b5ffb0, 0xd, 0x6fe8bf0, 0x0, 0x0, 0x0, 0x0)
	/gopath/src/github.com/gravitational/gravity/tool/gravity/cli/run.go:467 +0x68e7
github.com/gravitational/gravity/e/tool/gravity/cli.execute(0xc0001809a0, 0xc000b5ffb0, 0xd, 0x6fe8bf0, 0x0, 0x0, 0x0, 0x0)
	/gopath/src/github.com/gravitational/gravity/e/tool/gravity/cli/run.go:164 +0x5de
github.com/gravitational/gravity/e/tool/gravity/cli.Run(0xc0001809a0, 0xc0001809a0, 0xc000087b80)
	/gopath/src/github.com/gravitational/gravity/e/tool/gravity/cli/run.go:38 +0x3c7
main.run(0xc0008d82d0, 0x7, 0x3a8bfd7)
	/gopath/src/github.com/gravitational/gravity/e/tool/gravity/main.go:43 +0x39
main.main()
	/gopath/src/github.com/gravitational/gravity/e/tool/gravity/main.go:27 +0xfc
```

### Environment

 - Gravity version [e.g. 7.0.11]: 7.0.13

--- a/lib/ops/opsservice/service.go
+++ b/lib/ops/opsservice/service.go
@@ -1464,9 +1464,11 @@ func (o *Operator) EmitAuditEvent(ctx context.Context, req ops.AuditEventRequest
 		return trace.Wrap(err)
 	}
 	o.Infof("%s.", req)
-	err = o.cfg.AuditLog.EmitAuditEvent(req.Event, req.Fields)
-	if err != nil {
-		return trace.Wrap(err)
+	if o.cfg.AuditLog != nil {
+		err = o.cfg.AuditLog.EmitAuditEvent(req.Event, req.Fields)
+		if err != nil {
+			return trace.Wrap(err)
+		}
 	}
 	return nil
 }

